en/telemetry/esp8266_wifi_module.md

```diff
@@ -112,6 +136,22 @@ On your wifi-enabled *QGroundControl* ground station computer/tablet, find and c
 QGC automatically starts its UDP link on boot.
 Once your computer/tablet is connected to the **PixRacer** WiFi Access Point, it will automatically make the connection.
 
+## Configure QGC with non-standard Wifi connections
+
+QGC supports a variety of other wifi connections, if manually configured: 
+
+https://docs.qgroundcontrol.com/master/en/SettingsView/SettingsView.html
+
+If QGC doesn't automatically connect to your vehicle, add it's network connection as a custom comm link.
+
+1. Go to "Application Settings" > "Comm Links"
+       https://docs.qgroundcontrol.com/master/en/SettingsView/SettingsView.html
+2. add new one matching your settings above.
```

Comment:
You mean these ones? 
```
2. Factory Settings -- Example 2:

    - SSID: IFFRC_xxxxxxxx
    - Password: 12345678
    - IP: 192.168.4.1
    - Port: 6789 (TCP) 
 ```

Reply:
well, it might be these ones, it might be something different.... they're not really consistent across products :/ 

I ended up ordering both of the linked boards from banggood, before I figured out how it work.... and even those have a different back text.   Not just different SSID names, but they don't even have the same fields. :/ 

Honestly, it would hurt to add a "advanced users" tag to this page ....

Reply:
Thank you for the clarification. The important point is the one you already made - "check the box" :-)